index.rst

```diff
@@ -41,6 +41,12 @@ topics.
    dist_tables/postgresql_extensions.rst
 
 .. toctree::
+   :maxdepth: 1
+   :caption: Data Manipulation
```

Comment:
Instead of adding a new section (which we may also want to do at some point), we already cover data deletions in the append and [hash distribution sections of our docs](https://www.citusdata.com/docs/citus/5.1/dist_tables/append_distribution.html#dropping-shards).

Could we add this in those sections?

There are also certain statements there, which need to be updated, e.g. in the append-distribution page:
"Please note that this function only deletes complete shards and not individual rows from shards. If your use case requires deletion of individual rows in real-time, please consider using the hash distribution method."